### PR TITLE
ECA-signIn-signUp

### PR DESCRIPTION
**Pull request by edouardchop**

### CODE REVIEW

File: src/pages/sign-up/index.jsx [hunk 1]
```diff
@@ -38,37 +39,68 @@ const SignUpPage = () => {
         return
       }
 
-      router.push("/sign-in")
+      router.push("/")
```
Comment (TamWilliam) [inline src/pages/sign-up/index.jsx:42]: Why did you change the route to the homepage ?

**Review by TamWilliam — CHANGES_REQUESTED** [review]
Design seems good, but the route shouldn't be changed on signUp page.
Also you need to add the ticket number from Jira

**Review by TayZain — CHANGES_REQUESTED** [review]
Change the route of Sign Up
